app/src/main/java/org/oppia/android/app/home/promotedlist/PromotedStoryListViewModel.kt

```diff
@@ -28,6 +32,12 @@ class PromotedStoryListViewModel(
       activity.resources.getDimensionPixelSize(R.dimen.home_padding_end)
     else activity.resources.getDimensionPixelSize(R.dimen.home_padding_start)
 
+  /** Returns the type of PromotedActivity shown to the learner. */
+  fun getListType(): PromotedActivityType {
+    // TODO: Update this when more activity types are implemented.
+    return PromotedActivityType.RECENTLY_PLAYED
+  }
```

Comment:
Is this needed? Please remove this & the enum.

Reply:
Done